### DISCUSSION TITLE
Server is still in the process of starting | Connecting to a new peer

### DISCUSSION THREAD

**Discussion by Gchowdhury1988** [Ride-The-Lightning/RTL#857] [Troubleshooting Questions]
**Describe the bug**
I'm attempting to connect to my first peer.
 I've added some funds on chain. 
I enter the peers pubkey@ip:port info
I receive an error that says "server is still in the process of starting" 

This has been going on for maybe 30 minutes now. 
I'm running RTL over my Start9 Embassy. 

RTL is up to date.
I'm using Firefox browser and connecting to RTL over LAN.

**Comment by saubyk** [comment]
Hi @Gchowdhury1988 can you please confirm, if you're still facing this?

Also, would recommend that you should let your Bitcoin and Lightning node, sync completely before attempting any lightning operation.

**Reply by Gchowdhury1988** [reply]
> Hi @Gchowdhury1988 can you please confirm, if you're still facing this?
> 
> Also, would recommend that you should let your Bitcoin and Lightning node, sync completely before attempting any lightning operation.

I am not. I'm not sure what happened actually as I now have 4 peers identified and 1 channel opened. I didn't manually set any of this. I was only trying to connect with a peer who I didn't actually end up connecting with.

How did a channel open up? I had 800k sats, and now I have 476,961 on lightning.

**Reply by Gchowdhury1988** [reply]
FYI - this was solved with the start9 team. I installed LND in January and it had an autopilot setting on which opened the channels. Previously, the graph was not synced. Thank you.
